src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -97,6 +99,26 @@ namespace winrt::TerminalApp::implementation
         _filteredActionsView().SelectionChanged({ this, &CommandPalette::_selectedCommandChanged });
     }
 
+    // Method Description:
+    // - Attempts to load user locale, to further use it when sorting the actions in the filtered view
+    // Return Value:
+    // - The user level locale, upon failure returns a default locale
+    std::locale CommandPalette::_loadUserLocale()
+    {
+        // Loading user locale name explicitly using GetUserDefaultLocaleName,
+        // as std:locale("") crashes on some setups
+        wchar_t userLocaleName[LOCALE_NAME_MAX_LENGTH] = { 0 };
```

Comment:
nit: `wchar_t userLocaleName[LOCALE_NAME_MAX_LENGTH];`

is good enough I think!

Reply:
We need to explicitly null terminate it, because the next conversion function assumes that it receives a null terminated string. The solution I did here is for lazy ones, instead I should get the size from the function' return value and to put null char at this offset. I will fix it

Reply:
Ah I see, thank you!

Reply:
@PankajBhojwani - actually did a better approach. I call wstring ctor that gets the size. This should be both safe and efficient. Will submit shortly.